.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,281 @@
+name: e2e-tests
+
+on:
+  push:
+    tags:
+      - '*tetrate*'
+
+jobs:
+  create-test-images:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    name: create-test-images
+    runs-on: ${{ matrix.os }}
+    env:
+      HUB: hellozee-docker-istio-testing.bintray.io # change
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: docker-login
+        uses: docker/login-action@v1
+        with:
+          registry: hellozee-docker-istio-testing.bintray.io # change
+          username: ${{ secrets.BINTRAY_USER }}
+          password: ${{ secrets.BINTRAY_API_KEY }}
+
+      - name: build and push images
+        shell: 'script -q -e -c "bash {0}"'
+        run: |
+          export TAG=$(git describe --tags)
+          make docker push
+
+  eks-e2e-test:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+
+    name: eks-e2e-test
+    runs-on: ${{ matrix.os }}
+    needs: [create-test-images]
+    env:
+      HUB: hellozee-docker-istio-testing.bintray.io # change
```

Comment:
Can we set the TAG and HUB at the workflow level? For the release job we could overwrite.

Reply:
I did try it at first, didn't work, have to check the docs if I did that right or not